Index: Source/ThirdParty/ros/include/rcutils/error_handling.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Source/ThirdParty/ros/include/rcutils/error_handling.h b/Source/ThirdParty/ros/include/rcutils/error_handling.h
--- a/Source/ThirdParty/ros/include/rcutils/error_handling.h	(revision 584767ce5b22125a3cffe1e9e84618068f1431b4)
+++ b/Source/ThirdParty/ros/include/rcutils/error_handling.h	(date 1652721130821)
@@ -88,7 +88,7 @@
 } rcutils_error_state_t;
 
 // make sure our math is right...
-#if __STDC_VERSION__ >= 201112L
+//#if __STDC_VERSION__ >= 201112L
 static_assert(
   sizeof(rcutils_error_string_t) == (
     RCUTILS_ERROR_STATE_MESSAGE_MAX_LENGTH +
@@ -97,7 +97,7 @@
     RCUTILS_ERROR_FORMATTING_CHARACTERS +
     1 /* null terminating character */),
   "Maximum length calculations incorrect");
-#endif
+//#endif
 
 /// Forces initialization of thread-local storage if called in a newly created thread.
 /**
